frameworks/PHP/symfony/templates/bench/fortunes.html.twig

```diff
@@ -2,4 +2,11 @@
 
 {% block title "Fortunes" %}
 
-{% block content %}<table><tr><th>id</th><th>message</th></tr>{% for fortune in fortunes %}<tr><td>{{ fortune.id }}</td><td>{{ fortune.message }}</td></tr>{% endfor %}</table>{% endblock %}
```

Comment:
wasn't this done on purpose to save some bytes and make it faster?

Reply:
yes definitely, a micro optimization that one would never do in a production code...

on this dataset count that ~15 '\n' characters are 'overhead'

Reply:
now there are 9 '\n' in an output of 1204 chars, i think this is acceptable